### ISSUE TITLE
main window is grayed out when it lost its focus

### ISSUE THREAD

**Issue by gvdb1967** [zim-desktop-wiki/zim-desktop-wiki#1044]
zim 0.72 on fedora 31
After upgrading from 0.71 to 0.72, the notebook/notes window, main window is grayed out when it lost its focus. So it becomes less readable.

Is there a setting or place inside the code where this can be changed?

**Comment by sojusnik** [comment]
Same here on Ubuntu 19.10 with Gnome 3.34.

Would prefer a grayed out header bar, when unfocussed, with an unaltered text. See Gedit as an example.

**Comment by introt** [comment]
@sojusnik @gvdb1967 are you still having this issue with Zim 0.74?

**Comment by sojusnik** [comment]
@introt
Yes, especially annoying, when using Zim's main view with separate windows. It doesn't happen with all GTK apps, so it should be tweakable.

**Comment by jaap-karssenberg** [comment]
I really think this is Gtk3 behaviour. Probably it is theme-able, but not
sure how.

-- Jaap

Op wo 15 dec. 2021 12:11 schreef sojusnik ***@***.***>:
